weedcoco/tests/importers/test_mask.py

```diff
@@ -5,6 +5,13 @@
 
 from weedcoco.importers.mask import main
 
+# TODO: Further testing plan:
+# * check segmentation RLE string can be read back in and reproduces the mask
+# * check handling of missing correspondence between mask and image files
+# * check handling of different image file formats
+# * check handling of agcontext
```

Comment:
This would be good. The agcontext ingestion + testing would probably make sense as something that is shared as a utility called by each individual converter.